packages/radix-vue/src/DateField/DateFieldRoot.vue

```diff
@@ -199,7 +199,7 @@ watch([modelValue, locale], ([_modelValue]) => {
   if (_modelValue !== undefined) {
     segmentValues.value = { ...syncSegmentValues({ value: _modelValue, formatter }) }
   }
-  else if (Object.values(segmentValues.value).every(value => value === null)) {
+  else if (Object.values(segmentValues.value).every(value => value === null) || modelValue === undefined) {
```

Comment:
Also, for consistency, this should probably be `_modelValue` rather than `modelValue`?